include/benchmark/benchmark.h

```diff
@@ -292,7 +292,8 @@ namespace benchmark {
 class BenchmarkReporter;
 
 BENCHMARK_EXPORT void Initialize(int* argc, char** argv,
-                                 void (*HelperPrinterf)() = NULL);
+                                 void (*HelperPrinterf)(FILE*,
+                                                        const char*) = NULL);
```

Comment:
This is a breaking change to the API. If needed, I think we can overload `Initialize` to accept either an argumentless function or a two-argument version. Please say so if this is necessary.